src/staging/src/styles/parks.scss

```diff
@@ -2,13 +2,17 @@
 @import url("https://js.arcgis.com/4.10/esri/css/main.css");
 // @import url("https://governmentofbc.maps.arcgis.com/apps/webappviewer/jimu.js/css/jimu-theme.css");
 
-$mediumBreakpoint: 991.98px;
-
 .parks-container {
   //Added 50px extra to compensate padding
   max-width: 1370px !important;
   font-size: 15px;
   padding: 0px !important;
+  &.main-container {
+    padding-bottom: 0px !important;
+    @media (min-width: $lgBreakpoint) {
+      padding-bottom: 250px !important;
+    }
+  }
```

Comment:
The real issue is that the last section needs to be a certain height to trigger the scroll spy.

So I'd do something like: 
`.makeStyles-parkContent-2 > div:last-of-type {
    min-height: 600px;
}`

Though you may want to put a unique class on to that container as I don't know if `.makeStyles-parkContent-2` is unique.

But I'll also note that this issue exists on content pages as well, and I'll note that there seems to be an empty div at the end of some (all?) of the `.page-content` parent divs.

Reply:
FYI - I chose 600px as that'll trigger the final title if the display is 1080px high. That'll cover 97%+ of the cases. More looks goofy.

Reply:
Thanks! `.makeStyles-parkContent-2 > div:last-of-type { min-height: 600px; }` works perfectly 👏
I checked `.makeStyles-parkContent-2` is used in `park.js` and `site.js`, and it's okay since both templates are the same layout.

For the content page, let me work on the same issue in [this ticket]( https://bcparksdigital.atlassian.net/browse/CM-91) 🙂